package.json

```diff
@@ -35,8 +35,8 @@
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-import": "^15.1.0",
     "postcss-preset-env": "^8.4.2",
-    "vite": "^5.4.9",
-    "vite-plugin-ruby": "^5.1.1"
+    "vite": "^5.0.0",
```

Comment:
Why vite is downgraded ?

Reply:
I remember facing some issue for which i changed the version of vite and its ruby plugin ( I am unable to find that issue in my notes).
Lemme perform tests with the upgraded versions of vite and I ll let you know.